### ISSUE TITLE
mydumper creates insert statement without values

### ISSUE THREAD

**Issue by kfi** [xelabs/go-mydumper#15]
One file was created for this table, which has 6 INSERT statements. 5 are valid and were imported, the 6th doesn't have values and throws an sql error.

mydb.m2epro_request_pending_partial_data.00001.sql

```sql
INSERT INTO `m2epro_request_pending_partial_data`(`id`,`request_pending_partial_id`,`part_number`,`data`) VALUES
 ;
```

**Comment by BohuTANG** [comment]
Can you give the table schema and mydb.m2epro_request_pending_partial_data.00001.sql?

**Comment by kfi** [comment]
Hi, i sent it to you via email. I can't post the content public.

**Comment by BohuTANG** [comment]
@kfi 

Hi,
Fixed e66876cac330204c20431843338e757f89b2b2fc
Please pull from master and re-build again.

**Comment by kfi** [comment]
works, thank you!
